src/components/Markdown.svelte

```diff
@@ -1,9 +1,27 @@
 <script>
-  import { parseInline } from "marked";
+  // you would think we could just do:
+  // import marked from "marked", but that seems to cause
+  // weird problems with jest and storyshots (probably because
+  // of how it imports/compiles ES modules). this is the only
+  // workaround I found
```

Comment:
This makes me grumpy, but I couldn't find any alternative. :( I suspect this is an issue with marked (it came up earlier as well, though I can't find the reference)

Reply:
Bummer, but maybe a comment to that effect for future code readers is fine.